### ISSUE TITLE
The app does not install

### ISSUE THREAD

**Issue by papai12347** [MartyrPher/SMAPI-Android-Installer#46]

**Comment by MartyrPher** [comment]
The installer app or SMAPI app?

**Comment by leoguyman365** [comment]
I also aren't able to install it (the SMAPI app) and was wondering if you have an apk for the app directly because the installer keeps force closing

**Comment by JasonTodd25** [comment]
yeah mine too... it's force closed everytime i try to install SMAPI from the installer please fix this sir
